style.css

```diff
@@ -0,0 +1,218 @@
+html {
+	font-size: 10px;
+	scroll-behavior: smooth;
+	font-family: 'Varela Round', sans-serif;
+	box-sizing: border-box;
+}
+
+.nav-bar {
+	position: relative;
+	display:flex;
+	align-items: center;
+	justify-content: center;
+	gap: 30px;
+	background-color: #BD4B4B;
+	padding: 20px;
+	flex-wrap: wrap;
+}
+
+.nav-bar::after {
+ 	 position: absolute;
+	  bottom: -50px;
+ 	 left: 0px;
+ 	 width: 100%;
+	  content:" ";
+ 	 background:
+ 	 radial-gradient(circle at 50% 0%, #BD4B4B 25%, #EEEEEE 26%, #EEEEEE 40%);
+ 	 background-color: #EEEEEE ;
+ 	 background-size: 30px 100px;
+ 	 height: 50px;
```

Comment:
also this whole section is indented 1 space character more than the rest of the file